config/checkstyle_checks.xml

```diff
@@ -299,6 +299,9 @@
     <module name="VariableDeclarationUsageDistance"/>
 
     <!-- Filters-->
+    <module name="SuppressionXpathFilter">
+      <property name="file" value="${checkstyle.suppressions.file}"/>
```

Comment:
I assume since this is in the configuration, we now have XPath filter support?
If so, can you make atleast 1 modification to the suppression file to show it works?

Reply:
I assume this may require changing the DTD.
@romani I assume this is ok? Or do we want to hold off on this until a new issue?

Reply:
@timurt , please explain a reason of this update

Reply:
Otherwise I have following tests failed

> AllChecksTest.testAllCheckstyleModulesInCheckstyleConfig:443 checkstyle_checks.xml is missing module: SuppressionXpathFilter
> [ERROR]   AllChecksTest.testAllModulesAreReferencedInConfigFile:331->lambda$testAllModulesAreReferencedInConfigFile$1:334 SuppressionXpathFilter is not referenced in checkstyle_checks.xml

I can explicitly remove `SuppressionXpathFilter` from tests

Reply:
done